packages/react-instantsearch/src/core/createInstantSearch.js

```diff
@@ -25,6 +25,8 @@ export default function createInstantSearch(defaultAlgoliaClient, root) {
       createURL: PropTypes.func,
       searchState: PropTypes.object,
       onSearchStateChange: PropTypes.func,
+      onSearchParameters: PropTypes.func,
+      resultsState: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
```

Comment:
I'm not sure to get it, but it's basically depending if you're using it with mono or multi index.